### DISCUSSION TITLE
Running Sunshine on headless host

### DISCUSSION THREAD

**Discussion by testman42** [LizardByte/community#39] [Sunshine] [answered]
I have a headless computer that I use as a server for various things.

It has much more powerful CPU and GPU than my laptop.

That computer is running [Fedora Server](https://getfedora.org/en/server/), so it has no Xwindow or Wayland, it has no GUI at all, only command line (and Cockpit for remote)

Would it be possible to run Sunshine in some container (Docker, podman, etc.) where X is installed? 

Has anyone attempted anything like this already?

edit: changed topic from "remote host" to "headless host", as I realised that the first one was quite obvious. Of course Sunshine is meant to run on remote host, that's the point of it lol.

**Comment by ReenigneArcher** [comment, marked as answer]
We're planning to add officially supported docker images soon. In the meantime you can check out [Games on Whales](https://games-on-whales.github.io/gow/), which uses Sunshine.

**Reply by testman42** [reply]
Awesome, I was not aware of that project. Thank you for pointing it out. Will take a look at it, and will also keep an eye out for official docker image :+1:
